docs/cnv-configuration.md

```diff
@@ -0,0 +1,153 @@
+# Application Aware Quota
+## Motivation
+Managing VM Billing and Resource Quotas in a Multi-Tenant Cluster with Openshift-Virtualization
+
+Native ResourceQuotas in Openshift are designed to handle predefined pod compute resources such as
+`requests.cpu`, `limits.cpu`, `requests.memory`, `limits.memory`, and more. 
+
+Openshift-Virtualization allow the deployment and management of virtual machines alongside
+containerized applications.
+
+Administrators of multi-tenant clusters running Virtual Machines with OpenShift Virtualization 
+might expect tenants to be billed only for the resources explicitly requested in the VM object. 
+However, this presents specific challenges with resource quotas:
+
+1. VM Live Migration: VM instances run inside pods. Occasionally, the pod associated with a VM needs to
+   be evicted from one node and rescheduled onto another. VMs have a large state that requires careful
+   handling during migration. To manage this, a new pod is created to transfer the VM state before
+   terminating the old one, which differs from how standard pods are moved. Running both pods simultaneously
+   temporarily doubles the amount of resources the tenant is accounted for and can impact resource quotas,
+   potentially leading to migration issues and upgrade failures.
+2. Each Virtual Machine operates within its own container. This means that the virtualization overhead, 
+   runs within each VM pod instead of centrally on the host, as in traditional virtualization. 
+   This overhead is factored into quota calculations. In traditional virtualization platforms,
+   service providers typically base VM quotas solely on RAM and CPU threads, since users don't directly request
+   resources for this overhead or expect its effects. Moreover, the resource requirements for this overhead can
+   change between versions without users being notified.
```

Comment:
I would rewrite this to something like:
"Usually ResourceQuotas are used to bill the user for whatever he requested, hence should not account for the virtualization overhead which the user should not be aware of. Instead, the virtualization overhead is a non-transparent implementation detail that can change between versions."